pkg/private/zip/build_zip.py

```diff
@@ -142,6 +142,8 @@ def add_manifest_entry(self, entry):
 
     if entry_type == manifest.ENTRY_IS_FILE:
       entry_info.compress_type = zipfile.ZIP_DEFLATED
+      # Using utf-8 for the file names is for python <3.7 compatibility.
+      src = src.encode('utf-8')
       with open(src, 'rb') as src:
```

Comment:
Avoid aliasing and rename the second `src`?

Reply:
Done